src/webhook/config/manager/manager.yaml

```diff
@@ -27,41 +27,39 @@ spec:
       securityContext:
         runAsNonRoot: true
       containers:
-      - command:
-        - /manager
-        args:
-        - --leader-elect
-        image: controller:latest
-        name: manager
-        env:
-        - name: TARGET_RUNTIMECLASS
-          value: kata-remote
-        - name: POD_VM_INSTANCE_TYPE
-          value: t2.small
-        - name: POD_VM_EXTENDED_RESOURCE
-          value: kata.peerpods.io/vm
-        securityContext:
-          allowPrivilegeEscalation: false
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: 8081
-          initialDelaySeconds: 15
-          periodSeconds: 20
-        readinessProbe:
-          httpGet:
-            path: /readyz
-            port: 8081
-          initialDelaySeconds: 5
-          periodSeconds: 10
-        # TODO(user): Configure the resources accordingly based on the project requirements.
-        # More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
-        resources:
-          limits:
-            cpu: 500m
-            memory: 128Mi
-          requests:
-            cpu: 10m
-            memory: 64Mi
+        - command:
```

Comment:
All that region is reindented, which obfuscates the actual change, which I believe is only:

```diff
index 097df16..457a16c 100644
--- a/src/webhook/config/manager/manager.yaml
+++ b/src/webhook/config/manager/manager.yaml
@@ -36,8 +36,6 @@ spec:
           env:
             - name: TARGET_RUNTIMECLASS
               value: kata-remote
-        - name: POD_VM_INSTANCE_TYPE
-          value: t2.small
             - name: POD_VM_EXTENDED_RESOURCE
               value: kata.peerpods.io/vm
           securityContext:
```

Reply:
Sorry about it.. I'll redo these changes. I didn't realise my IDE auto-formatted it